app/views/maintenance_tasks/tasks/show.html.erb

```diff
@@ -4,7 +4,14 @@
   <%= @task %> <%= status_tag(@task.status) %>
 </h1>
 
-<%= render 'maintenance_tasks/runs/info', run: @task.last_run, with_status: false if @task.last_run %>
+<% if last_run = @task.last_run %>
+  <%= render 'maintenance_tasks/runs/info', run: last_run, with_status: false %>
```

Comment:
Do you think the link could go inside the `maintenance_tasks/runs/info` partial instead?